joomla/joomla-cms
# [4.1] Check File Size task options

The Directory to check is a required field. But because of the options in the xml it will happily store "None Selected" which is an error and so the required attribute does nothing.

This PR corrects that behaviour

To test
Create a new scheduled task for Image Size check. You can save the task with directory set as None Selected
After the pr you will now get an error if you try to save with None Selected


### After
![image](https://user-images.githubusercontent.com/1296369/144854381-d258dbed-a985-41c2-a3be-1dbf35656d6b.png)
